create-project-async/create_project_async.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import os
+import json
+import sys
+import glob
+from dotenv import load_dotenv
+from helper import get_access_token, get_operations
+
+os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
+
+load_dotenv()
+BASE_URL = os.getenv('BASE_URL')
+CLIENT_ID = os.getenv('CLIENT_ID')
+CLIENT_SECRET = os.getenv('CLIENT_SECRET')
+TEAM_ID = os.getenv('TEAM_ID')
+URL = BASE_URL + "/graphql"
+
+# Read Json payload from external file to make it more convenient
+operations = get_operations('create_project_async.json')
+
+# Set input.teamId
+operations["variables"]["input"]["teamId"] = str(TEAM_ID)
+
+payload_map = {}
+documents = []
+files = []
+
+# Set files in documents folder as input.documents
+iterator = 0
+for filepath in glob.iglob('documents/*.txt'):
```

Comment:
does it only work for .txt files?

or we can put comment around here to guide user that they can use any extension that they want

Reply:
ya, it should be 'documents/*'